src/traversal/writeRelayUpdatePayload.js

```diff
@@ -76,6 +76,13 @@ function writeRelayUpdatePayload(
   payload: {[key: string]: mixed},
   {configs, isOptimisticUpdate}: UpdateOptions
 ): void {
+  invariant(
+  typeof payload.clientMutationID === 'string',
+  'writeRelayUpdatePayload: expected clientMutationID on the payload for mutation %s to be string' +
+  ', instead received the %s %s',
+  operation.getCall().name, typeof payload.clientMutationID, payload.clientMutationID
+);
+
```

Comment:
the same applies here, but use `payload[CLIENT_MUTATION_ID]` - internally at FB the key is `clientMutationId`, in OSS it's `client_mutation_id`, and this property allows us to use the correct one for the environment